.github/workflows/preview.yml

```diff
@@ -32,6 +32,7 @@ jobs:
           rm spec/index.bs wgsl/index.bs explainer/index.bs
           wget -O spec/index.bs https://raw.githubusercontent.com/${{ github.event.workflow_run.head_repository.full_name }}/${{ github.event.workflow_run.head_sha }}/spec/index.bs
           wget -O wgsl/index.bs https://raw.githubusercontent.com/${{ github.event.workflow_run.head_repository.full_name }}/${{ github.event.workflow_run.head_sha }}/wgsl/index.bs
+          wget -O wgsl/extract-grammar.py https://raw.githubusercontent.com/${{ github.event.workflow_run.head_repository.full_name }}/${{ github.event.workflow_run.head_sha }}/wgsl/extract-grammar.py
```

Comment:
We intentionally avoid python scripts to be there (including IDL extractor) since it allows very easy access to some private keys.

Reply:
ah, bummer.

Reply:
Removed and added a comment

Reply:
@dj2 Thanks!